.travis.yml

```diff
@@ -6,25 +6,21 @@ matrix:
   fast_finish: true
   include:
     # macOS build
-    - env: INSTALL_TYPE=develop FITS="astropy" TEST=submodule MATPLOTLIBVER=2.0 XSPECVER="12.9.1" TRAVIS_PYTHON_VERSION="3.6"
+    - env: INSTALL_TYPE=develop FITS="astropy" TEST=submodule MATPLOTLIBVER=2 XSPECVER="12.9.1" TRAVIS_PYTHON_VERSION="3.6"
       os: osx
     # Barebone build to check tests pass when most of the tests must be skipped.
     # We need to use TEST=none to remove the test-data submodule.
     - env: INSTALL_TYPE=develop TEST=none NUMPYVER=1.11 TRAVIS_PYTHON_VERSION="2.7"
-    # Full build (including ds9 and xspec), Python 2.7, setup.py develop, astropy, matplotlib 1.5
-    - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1.5 TRAVIS_PYTHON_VERSION="2.7"
+    # Full build (including ds9 and xspec), Python 3.5, setup.py develop, astropy, matplotlib 1.5
+    - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1 TRAVIS_PYTHON_VERSION="3.5"
       sudo: required
       dist: trusty
-    # As above, Python 3.5
-    - env: XSPECVER="12.9.1" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER=1.5 TRAVIS_PYTHON_VERSION="3.5"
+    # As above, Python 3.6, setup.py install, xspec 12.10
+    - env: XSPECVER="12.10.0e" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2 TRAVIS_PYTHON_VERSION="3.6"
```

Comment:
I've noticed that the `INSTALL_TYPE` here has been changed from `develop` to `install`. I don't have a problem with this change but I just wanted to check it was intended.

Reply:
That's fine. It's a way to explore more of the infinite number of possible combinations in which Sherpa can be built.